packages/zowe-explorer-api/src/extend/MainframeInteraction.ts

```diff
@@ -506,7 +516,7 @@ export namespace MainframeInteraction {
          * @param {string} command
          * @param {string} cwd
          * @param {boolean} flag
-         * @returns {string>}
+         * @returns {Promise<string>}
```

Comment:
Should we update the other `issueXyzCommand` interfaces above?

I believe they all return a promise 😋

Reply:
Probably, I just noticed this while adding the `uploadFromBuffer` definition - addressed in b5d9d3a1 😋